### PR TITLE
Add cla checks

### PR DESCRIPTION
**Pull request by psdhanesh7**
Add CLA checks to this repository.
Solve task 2 in issue https://github.com/brackets-cont/contributor-license-agreement/issues/22#issue-1221753274.

Modifications done in the already existing .github/workflows/python/validate_cla_signature.py file to use the requests library instead of custom code.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -63,3 +63,6 @@ package-lock.json
 
 # Files build by scripts
 /src/thirdparty/semver.browser.js
+
+# Temporary files for testing CLA checks
```
Comment (abose) [inline .gitignore:67]: What is this for?

Comment (psdhanesh7) [reply .gitignore:67]: .tmp folder was created in the local environment to simulate the github environment for some local testing purposes.

**Review by abose — APPROVED** [review]
LGTM

**Comment by abose** [comment]
Thanks for the changes @psdhanesh7 
Merging

**Pull request merged**